### ISSUE TITLE
Cookie banner obscures navigation

### ISSUE THREAD

**Issue by sftim** [crossplane/crossplane.github.io#162]
The cookie banner at https://crossplane.io/ obscures the site navigation.

I'm reporting a near-duplicate of #108 because the sites at https://crossplane.github.io/ and https://crossplane.io/ _do_ have cookie banners still.

**Comment by plumbis** [comment]
I apologize, you are correct, I was wrong! The cookie banner external resource was being blocked by my DNS (pihole).

I'll look into the issue and see what we can do.

**Comment by plumbis** [comment]
This is from the Hubspot JS include- https://github.com/crossplane/crossplane.github.io/blob/master/themes/crossplane/layouts/partials/hubspot.html

**Comment by plumbis** [comment]
Poking at this made me realize that the problem isn't the banner but it's the fact you can't decline. The banner is supposed to be obnoxious to force a cookies choice. A fix was pushed at the tracker level to render the a decline button.

Does this work for you @sftim ?

**Comment by sftim** [comment]
Cool, being able to decline cookies fixes this for me. I can do that.
